src/api/MainSolver.cc

```diff
@@ -492,12 +445,10 @@ MainSolver::SubstitutionResult MainSolver::computeSubstitutions(PTRef fla) {
         PTRef simp_formula = root;
         // l_True : exists and is valid
         // l_False : exists but has been disabled to break symmetries
-        MapWithKeys<PTRef,lbool,PTRefHash> new_units;
+        MapWithKeys<PTRef, lbool, PTRefHash> new_units;
         vec<PtAsgn> current_units_vec;
         bool rval = logic.getNewFacts(simp_formula, new_units);
-        if (not rval) {
-            return SubstitutionResult{{}, logic.getTerm_false()};
-        }
+        if (not rval) { return SubstitutionResult{{}, logic.getTerm_false()}; }
```

Comment:
I think the former format was better

Reply:
This is the style we eventually agreed on. For one-line if-blocks, we put them on the same line.

Reply:
Since it is a block, it does not make too much sense to me.
In the case of one-liners, only one command should be used, so I would keep it this way but remove the block `{}`.
In the rest cases, I would use the multiline variant.